src/dropbox.js

```diff
@@ -66,6 +66,24 @@ export default class Dropbox {
   }
 
   request(path, args, auth, host, style) {
+    let multiAuth;
+    if (auth.split(',').length > 1) {
+      // if there are multiple auth types, creates set with different auth types
+      const authTypes = auth.replace(' ', '').split(',');
+      multiAuth = new Set(authTypes);
+    }
+    // checks which auth types are in the multiauth and assigns one based on priority
+    // to create header in switch case
+    if (multiAuth) {
+      if (multiAuth.has(USER_AUTH) && this.auth.getAccessToken()) {
```

Comment:
Out of curiosity, how did we arrive to this order of priority? Would this cause any issues if the multiauth type was say `user,team` and they only had a team token? Since get access token wouldn't cover that here?